apps/wing/src/commands/test/snapshots-help.ts

```diff
@@ -0,0 +1,17 @@
+export const SNAPSHOTS_HELP = `
+Snapshots (s, --snapshots <mode>):
+  never  \t Snapshot are never captured
+  auto   \t Determines behavior based on the "CI" environment variable: "assert" if CI=1 or "deploy" otherwise
+  deploy  \t Execute tests on the target platform and update snapshots if all tests pass
+  assert \t verifies that the snapshot is up-to-date and fails the test is they are not
+  update \t Only update the snapshots without actually executing the tests on the target platform
+
+  When testing against a cloud target (e.g. -t tf-aws), if all tests pass, the compiler output 
+  will be captured under "<entrypoint>.snap.md".
+`;
```

Comment:
I don't think we need the initial newline. Also, it looks like we can get rid of an extra space before one of the tabs. I didn't test locally though so I can't say for sure.

```suggestion
export const SNAPSHOTS_HELP = `\
Snapshots (s, --snapshots <mode>):
  never  \t Snapshot are never captured
  auto   \t Determines behavior based on the "CI" environment variable: "assert" if CI=1 or "deploy" otherwise
  deploy \t Execute tests on the target platform and update snapshots if all tests pass
  assert \t verifies that the snapshot is up-to-date and fails the test is they are not
  update \t Only update the snapshots without actually executing the tests on the target platform

  When testing against a cloud target (e.g. -t tf-aws), if all tests pass, the compiler output 
  will be captured under "<entrypoint>.snap.md".
`;
```